src/quo2/components/tags/network_tags/style.cljs

```diff
@@ -0,0 +1,35 @@
+(ns quo2.components.tags.network-tags.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn network-tags-container
+  [status]
+  {:flex-direction   :row
+   :background-color (if (= status "Error")
```

Comment:
Probably this should be a keyword not a string

Reply:
Done!